app/main.py

```diff
@@ -1,3 +1,30 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from typing import Callable
+
+from app.shop import Shop
+from app.customer import Customer
+
+
+def get_data(info_key: str, creation_func: Callable) -> list:
+    with open("app/config.json", "r") as source:
```

Comment:
Ensure that the path to the JSON file 'app/config.json' is correct and that the file exists in the specified location.